### DISCUSSION TITLE
Plugins not found in venv. Fresh install

### DISCUSSION THREAD

**Discussion by Starkbock** [netbox-community/netbox#14268] [Help Wanted!]
Hello all
I just installed a fresh unbunt and then followed the install guide for netbox. All is dandy that far. Now I want to load a few plugins and this is were i fail.

When I pip install a pugin, for example netbox-qrcode (cause its less config for it) it downloads and installs it. I am in the netbox venv and it seems to go fine. If I run pip status netbox-qrcode it displays the plugin. 

As soon as I add the PLUGIN = ['netbox_qrcode'] I get the Bad gateway 502. 

Doing the journalctl -eu netbox/nginx (cant remeber where i said it) it says it cant find the plugin.

Since my install is 16 hours old and by the book - Is the bug or something I hit? Or is it a rights issue? When i install in the venv i use sudo pip, which maybe is problematic?

Hope it makes sense.. 

Regards
Patrik

**Comment by abhi1693** [comment]
Can you all relevant logs for the steps that you have mentioned?

**Comment by Starkbock** [comment]
Figured it out.
You have to activate the venv as root (sudo -do root source
/opt/netbox/venv/bin/activate) then run pip install <netbox-pluginhere>
Else it wont work..
5 days.. Am better at windows =P
